common.lic

```diff
@@ -268,9 +268,10 @@ module DRC
     end
   end
 
-  def hide?
+  def hide?(*hide_type)
+    hide_type.any? ? hide_type = hide_type[0] : hide_type = 'hide' 
```

Comment:
Wasn't sure what to put here since these all seem to be lists

base-empty.yaml

  gear_sets: {}
  lichbot_buffs: []
  listen_skills: []
  offensive_spell_cycle: []
  ignored_npcs: []
  hunting_buddies: []
  empty_hunting_room_messages: []
  classes_to_teach: []
  hide_type:

and for base.yaml
hide_type:
or 
hide_type: hide